scripts/config.py

```diff
@@ -358,6 +358,33 @@ def continuation(name, active, section):
         return adapter(name, active, section)
     return continuation
 
+PLATFORM = frozenset([
+    'MBEDTLS_PLATFORM_C',
```

Comment:
Why are these enumerated instead of disabling all `MBEDTLS_PLATFORM_.*`?

Reply:
Because I thought it might be clearer, and the annoyance with `MBEDTLS_ENTROPY_NV_SEED`, but sure I can pattern match instead